src/main/java/frc/robot/subsystems/vision/NoteCamera.java

```diff
@@ -0,0 +1,50 @@
+package frc.robot.subsystems.vision;
+
+
+import org.littletonrobotics.junction.Logger;
+import org.photonvision.PhotonCamera;
+import org.photonvision.targeting.PhotonTrackedTarget;
+
+public class NoteCamera {
+
+  PhotonCamera camera;
+  String path;
+
+  private final double MIN_AREA = 10;
+
+  public NoteCamera(String name) {
+    // This is a constructor
+    camera = new PhotonCamera(name);
+    path = "Vision/" + name.replace("_", "");
+  }
+
+  public PhotonTrackedTarget getTarget() {
+    if(1==1) return null;
+    var res = camera.getLatestResult();
+    int numTargets = res.getTargets().size();
+    Logger.recordOutput(path + "/Valid", numTargets > 0);
+    Logger.recordOutput(path + "/NumTargets", numTargets);
+
+    var best = numTargets > 0 ? res.getTargets().get(0) : null;
+
+    if(best != null) {
```

Comment:
use Object.isNull(best) instead

Reply:
So, I was curious what that method was meant for, and apparently it was added specifically for Java 8's streams, so that you can do stuff like `.stream().filter(Objects::isNull)`. While that'd work just fine, I don't see the point (it is quite literally a wrapper around `if (val == null)` and reads the same in my head)

Reply:
It's a standard in programming to replace != null with that method or a similar one - it addresses readability and usability standards. Basically it makes the code more readable and avoids potential typos and assignment to val. Though it can boil down to style preference